### DISCUSSION TITLE
Supabase policies not recognizing query from logged in user

### DISCUSSION THREAD

**Discussion by akhan280** [supabase/supabase#22068] [Questions]
I'm making queries to my DB from an authenticated user (set in cookies). I've set some very simple policies

<img width="393" alt="Screenshot 2024-03-16 at 9 15 01 PM" src="https://github.com/supabase/supabase/assets/35405856/c1669063-7817-4700-a72a-0b14f28d8678">

when making my SB query, do I need to pass in my auth token? this is how i'm doing my queries:
  const { error } = await supabase
    .from('users')
    .upsert({
      phone_number: `${phone_number}`,
      reportDone: true,
      report: `${editableText}`,
      report_date_submitted: new Date(),
  }, {
      onConflict: 'phone_number',
  })
  .select();
  
 I haven't seen any documentation for this, so if anyone has links I would greatly appreciate it

**Comment by GaryAustin1** [comment]
You asked the same question on Discord.  
Not enough info provided to answer clearly.
Without the actually policies, will assume they are valid...

You don't say your framework or if you are using auth-helpers/ssr.  Those handle server side coordination of browser/server clients with cookies.

If you are doing your own code on the server and passing the session in a cookie then yes you need to use the global header option in createClient to set the authorization header to 'bearer userjwt'.
